src/molecules/form/component.tsx

```diff
@@ -25,7 +25,7 @@ const LoadingOverlay = styled.div`
   position: absolute;
   top: 0;
   left: 0;
-  background-color: rgba(0, 0, 0, 0.7);
+  background-color: rgba(0, 0, 0, 0.05);
```

Comment:
this should not be black it should use the background color

Reply:
like it should be customization from the theme

Reply:
well it can either be added to the form theme or the background color from the theme.colors

Reply:
probably better is to add a overlayColor to the form theme and then set this to have a sensible default